projects/collection-editor-library/src/lib/components/options/options.component.ts

```diff
@@ -139,7 +139,7 @@ export class OptionsComponent implements OnInit {
 
   setScore(value, scoreIndex) {
     const obj = {
-      respone: scoreIndex,
+      response: scoreIndex,
```

Comment:
did we handle this in all places?

Reply:
spelling is correct no other place we are using it.